Documentation/tutorials/data-prep-tutorial.md

```diff
@@ -0,0 +1,210 @@
+# Data Prep for Data Collection <!-- omit in toc -->
```

Comment:
- Map track event data with Data Prep for Data Collection
- Also don't forget to include it in the tutorials/README toc